recipes/mpl-probscale/meta.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: mpl-probscale
+  version: 0.1.1
+
+source:
+  git_url: https://github.com/phobson/mpl-probscale.git
+  git_tag: v0.1.1
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+build:
+  number: 1
+  script: python setup.py install
```

Comment:
That is the latest conda-build. You can use it everywhere. You can even do multiline commands like [this](https://github.com/conda-forge/ipykernel-feedstock/pull/2/files).